zyte_api/aio/client.py

```diff
@@ -29,7 +29,8 @@ def create_session(connection_pool_size=100, **kwargs) -> aiohttp.ClientSession:
     """ Create a session with parameters suited for Zyte API """
     kwargs.setdefault('timeout', AIO_API_TIMEOUT)
     if "connector" not in kwargs:
-        kwargs["connector"] = TCPConnector(limit=connection_pool_size)
+        kwargs["connector"] = TCPConnector(limit=connection_pool_size,
+                                           force_close=True)
```

Comment:
I assume the reconnect overhead should not be a problem given the overall response time if the ping is good.
It might probably be in seconds for some percentile since it should (AFAIR) 3xRTT time.

Do we need a follow-up on that to try to optimise in in the future?

Reply:
Yes, I think it'd be nice to get to the root of it.
Probably it would make sense to focus on http2 support in the client though, not on http 1.1 keep-alive. It would require switching to a different http library.